posthog/api/session_recording.py

```diff
@@ -136,19 +138,51 @@ def delete(self, request: request.Request, *args: Any, **kwargs: Any) -> Respons
 
         return Response({"success": True})
 
+    @action(methods=["GET"], detail=True)
+    def snapshot_file(self, request: request.Request, **kwargs) -> HttpResponse:
+        blob_key = request.GET.get("blob_key")
+
+        if not blob_key:
+            raise exceptions.ValidationError("Must provide a snapshot file blob key")
+
+        # very short-lived pre-signed URL
+        file_key = f"session_recordings/team_id/{self.team.pk}/session_id/{self.kwargs['pk']}/{blob_key}"
```

Comment:
At some point we should validate this pk value to be a UUID. Not yet but at some point